cli/internal/helm/suite_install.go

```diff
@@ -0,0 +1,268 @@
+	if err != nil {
+		return fmt.Errorf("setting up microservice values: %w", err)
+	}
+	if err := h.installer.InstallChartWithValues(ctx, releases.ConstellationServices, serviceVals); err != nil {
+		return fmt.Errorf("installing microservices: %w", err)
+	}
+
+	h.log.Debugf("Installing cert-manager")
+	if err := h.installer.InstallChart(ctx, releases.CertManager); err != nil {
+		return fmt.Errorf("installing cert-manager: %w", err)
+	}
+
+	if releases.CSI != nil {
+		var csiVals map[string]any
+		if provider == cloudprovider.OpenStack {
+			creds, err := openstack.AccountKeyFromURI(serviceAccURI)
+			if err != nil {
+				return err
+			}
+			cinderIni := creds.CloudINI().CinderCSIConfiguration()
+			csiVals = map[string]any{
+				"cinder-config": map[string]any{
+					"secretData": cinderIni,
+				},
+			}
+		}
+
+		h.log.Debugf("Installing CSI deployments")
+		if err := h.installer.InstallChartWithValues(ctx, *releases.CSI, csiVals); err != nil {
+			return fmt.Errorf("installing CSI snapshot CRDs: %w", err)
+		}
+	}
+
+	if releases.AWSLoadBalancerController != nil {
+		h.log.Debugf("Installing AWS Load Balancer Controller")
+		if err := h.installer.InstallChart(ctx, *releases.AWSLoadBalancerController); err != nil {
+			return fmt.Errorf("installing AWS Load Balancer Controller: %w", err)
+		}
+	}
+
+	h.log.Debugf("Installing constellation operators")
+	operatorVals := setupOperatorVals(ctx, idFile.UID)
+	if err := h.installer.InstallChartWithValues(ctx, releases.ConstellationOperators, operatorVals); err != nil {
+		return fmt.Errorf("installing constellation operators: %w", err)
+	}
+	return nil
+}
+
+type helmInstaller interface {
+	InstallChart(context.Context, Release) error
+	InstallChartWithValues(ctx context.Context, release Release, extraValues map[string]any) error
+}
+
+// lb_port: everywhere hardcoded in TF 6443, for host use ip
+// TODO(malt3): switch over to DNS name on AWS and Azure
+// soon as every apiserver certificate of every control-plane node
+// has the dns endpoint in its SAN list.
+func setupCiliumVals(ctx context.Context, provider cloudprovider.Provider, dsClient *k8sDsClient, output terraform.ApplyOutput) (map[string]any, error) {
+	vals := map[string]any{
+		"k8sServiceHost": output.IP,
+		"k8sServicePort": 6443, // TODO take from tf?
+	}
+
+	// GCP requires extra configuration for Cilium
+	firstNodePodCIDR := "10.10.1.0/24" // TODO how to get this?
+	if provider == cloudprovider.GCP {
+		// TODO(elchead): remove?
+		if err := dsClient.PatchNode(ctx, firstNodePodCIDR); err != nil {
+			return nil, fmt.Errorf("patching GCP node: %w", err)
+		}
+		vals["ipv4NativeRoutingCIDR"] = output.GCP.IPCidrPod
+		vals["strictModeCIDR"] = output.GCP.IPCidrPod
+	}
+	return vals, nil
+}
+
+// setupMicroserviceVals returns the values for the microservice chart.
+func setupMicroserviceVals(provider cloudprovider.Provider, measurementSalt []byte, uid, serviceAccURI string, output terraform.ApplyOutput) (map[string]any, error) {
```

Comment:
Is there a reason why this couldn't be integrated with `extendConstellationServicesValues`?

Reply:
no, we could unify all chart setup now, it's a good idea. But would do so in a separate PR